resource/src/main/java/za/co/sfy/lucid/sams/resource/CycleInfoResource.java

```diff
@@ -0,0 +1,62 @@
+package za.co.sfy.lucid.sams.resource;
+
+import org.springframework.stereotype.Component;
+import za.co.sfy.lucid.sams.resource.connection.DatabaseConnectionManager;
+import za.co.sfy.sams.lucid.schema.CycleInfo;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+
+/**
+ * @author muzim
+ */
+@Component
+public class CycleInfoResource  extends AbstractLucidSAMSResource implements ILucidSAMSResource {
+
+    private final String TABLE_NAME = "Cycleinfo";
```

Comment:
Please also just double check table names. I see some inconsistencies. but it might be on there side. I cant remember if they stick to some sort of naming convention on their side.